src/proxies/panel_manager.js

```diff
@@ -15,7 +15,7 @@ PanelManager.setPoi = async function(poi, options) {
   __panel_manager.panels.forEach((panel) => {
     if(panel.isPoiComplient) {
       panel.setPoi(poi, options)
-    } else {
+    } else if(options && !options.isFromList && !options.isFromFavorite){
       panel.close()
```

Comment:
without this test, the PJ PoIs disappear and the OSM PoIs reappear when clicking a PJ list item

Reply:
would you like to add a parameter to closePanel to avoid switching PJ and OSM layers, or something like that? I thought of making that but I'm not comfortable enough with the "code base" to be sure of all the implications it could have.